.eleventy.js

```diff
@@ -25,19 +25,27 @@ require("./src/js/utils.js");
 
 /**
  * Get an array of unique tags from a collection.
+ * Note: tags with the same spelling but different case usage will each be considered unique.
  *
- * @param {Object} collection An Eleventy collection defined via the collections API: https://www.11ty.dev/docs/collections/#advanced-custom-filtering-and-sorting
- * @returns {Array}
+ * @param {Object} collection - An Eleventy collection defined via the collections API: https://www.11ty.dev/docs/collections/#advanced-custom-filtering-and-sorting
+ * 
+ * @returns {Array<Object>} An array of tag objects with the string properties `name` and `slug`
  */
 const getUniqueTags = function(collection) {
-	const tagsSet = new Set();
+	let tagsMap = new Map();
 	collection.forEach(item => {
 		if (!item.data.tags) return;
 		item.data.tags
 			.filter(tag => !["pages", "initiatives", "news", "views", "comments"].includes(tag))
-			.forEach(tag => tagsSet.add(tag));
+			.forEach(tag => {
+				if (!tagsMap.has(tag)) {
+					tagsMap.set(tag, slugFilter(tag));
```

Comment:
Using slugified tags as map keys might be a better option since the slug function will lower case all letters. This will make sure all case related handling afterwards can be removed. Line 41-43 looks like:

```
let slugifiedTag = slugFilter(tag);
if (!tagsMap.has(slugifiedTag)) {
	tagsMap.set(slugFilter(tag), tag);
}
```

Moreover, the sort at line 48 could sort by case-insensitive slugs as well.